<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuCreatePlaylistDlgUI</class>
 <widget class="QWidget" name="MuCreatePlaylistDlgUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>New PlayList</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="titleLE">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border: 1px solid lightgray;
	border-radius: 1px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Enter the new play list title</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="createBtn">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#createBtn::checked {
        border: 1px solid rgb(17, 103, 168);
        background-color: rgb(17, 103, 168);
        border-radius: 4px;
        color: white;
}
QPushButton#createBtn::checked:hover {
        border: 1px solid rgb(13, 83, 134);
        background-color: rgb(13, 83, 134);
        border-radius: 4px;
        color: white;
}
QPushButton#createBtn {
        background-color: rgb(150, 192, 225);
        border: 1px solid rgb(150, 192, 225);
        border-radius: 4px;
        color: lightgray;
}</string>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#cancelBtn {
        background-color: white;
        border: 1px solid lightgray;
        border-radius: 4px;
        color: black;
}

QPushButton#cancelBtn:hover {
        background-color: rgb(239, 239, 239);
        border: 1px solid lightgray;
        border-radius: 4px;
        color: black;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
